src/App.tsx

```diff
@@ -1,18 +1,50 @@
 import React from 'react';
 
+import 'bootstrap/dist/css/bootstrap.min.css';
 import './App.scss';
 
+import { Tabs } from './components/Tabs/Tabs';
+
 const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+type Props = {};
+
+type State = {
+  tabsFromServer: Tab[];
+  selectedTab: Tab;
+};
+
+class App extends React.Component<Props, State> {
+  state: State = {
+    tabsFromServer: tabs,
```

Comment:
avoid `fromServer`, it doesn't matter here
```suggestion
    tabs: tabs,
```

Reply:
> avoid `fromServer`, it doesn't matter here

Great point, thanks ;)